README.md

```diff
@@ -1,15 +1,26 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/FgMJElkj)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. Asymptotic analysis is concerned with the performance of an algorithm with very large inputs. In practice input sizes can vary, therefor the analysis may differ. An analysis of the time complexity at a smaller input may be drastically different from the asymptotic complexity, and smaller inputs may be what an algorithm is concerned with practically. 
```

Comment:
Take an algorithm with an asymptotic complexity of $n^2$. With very small inputs, this algorithm may run very fast, maybe even as fast as algorithms with a better asymptotic complexity. This can be misleading because $n^2$ grows much faster than non quadratic asymptotics at larger inputs, but with smaller inputs it could perform its function relatively fast.

Reply:
How is this misleading? This is what the fact that the runtime is quadratic tells you.